cmd/tools/generate/storagegrid_counter.tmpl

```diff
@@ -0,0 +1,64 @@
+This document describes how Harvest metrics relate to their relevant StorageGrid, including:
```

Comment:
This document describes which StorageGRID metrics are collected and what those metrics are named in Harvest, including: